code/modules/projectiles/ammunition/magazines.dm

```diff
@@ -158,6 +158,12 @@
 /obj/item/ammo_box/magazine/internal/shot/toy/crossbow
 	max_ammo = 5
 
+//crossbow
+/obj/item/ammo_box/magazine/internal/shot/crossbow
+	ammo_type = /obj/item/ammo_casing/caseless/crossbowbolt && /obj/item/ammo_casing/apfsdsbolt

```

Comment:
Hilariously this compiles
Presumably it's constant folding which means you've done `ammo_type = 1`